app/forms/waste_carriers_engine/renewal_received_form.rb

```diff
@@ -10,10 +10,10 @@ def self.can_navigate_flexibly?
 
     def initialize(transient_registration)
       super
-      self.contact_email = @transient_registration.contact_email
-      self.pending_convictions_check = @transient_registration.conviction_check_required?
-      self.pending_payment = @transient_registration.pending_payment?
-      self.pending_worldpay_payment = @transient_registration.pending_worldpay_payment?
+      self.contact_email = transient_registration.contact_email
```

Comment:
NIT Adding a line between super and this as per other changes

```suggestion
      
      self.contact_email = transient_registration.contact_email
```